Dockerfile

```diff
@@ -170,7 +170,7 @@ RUN /bin/bash /tmp/MLOS/scripts/install.python.sh && \
 
 RUN python3.7 -m pip install pip && \
     python3.7 -m pip install --upgrade pip && \
-    python3.7 -m pip install setuptools wheel
+    python3.7 -m pip install setuptools wheel pytest
```

Comment:
Can you please add pylint here as well?  I hadn't realized we didn't include it yet (probably already in the base somehow).